composer.json

```diff
@@ -19,6 +19,9 @@
     "simshaun/recurr": "^5",
     "ipl/stdlib": ">=0.12.0"
   },
+  "suggest": {
+    "ext-ev": "Allows to have over 1024 open file descriptors in parallel. Additionally, it improves performance with just a handful of FDs, and the benefit grows with each additional connection."
```

Comment:
> each additional *connection*

The audience of this message are users of this library, right? Can we provide a message then that doesn't explain an implementation detail?

Reply:
I don't get what is supposed to be implementation details in the quoted words.

Reply:
This library is used to schedule jobs. How it does that, is an implementation detail. Whether it uses syscalls or it's own timer implementation is irrelevant to the user of it, or not?

Reply:
At least I consider the information about 1024 maximum parallel connections worthwhile.

Reply:
You should really get into the habit of explaining your reasoning :wink:

Reply:
Which kind of explanation should we give to the user then, when we don't mention the actual reason for installing it?

Reply:
*Improves performance, efficiency and avoids system limitations. Highly recommended! (See $link for details)*